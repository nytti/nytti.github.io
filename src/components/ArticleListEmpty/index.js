// Stateless component to show empty list of articles
import React from 'react'

// Material UI
import Typography from '@material-ui/core/Typography'

export default function ArticleListEmpty() {
  return (
    <Typography variant="headline">
      Nothing found...
    </Typography>
  )
}
