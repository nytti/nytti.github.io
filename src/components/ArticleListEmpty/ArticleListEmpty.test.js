import React from 'react'
import { shallow } from 'enzyme'

import ArticleListEmpty from './index'

describe('<ArticleListEmpty />', () => {
  test('<ArticleListEmpty /> snapshot', () => {
    const element = shallow(<ArticleListEmpty />)
    expect(element).toMatchSnapshot()
  })
})
