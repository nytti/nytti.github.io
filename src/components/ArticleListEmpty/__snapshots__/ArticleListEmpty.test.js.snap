// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ArticleListEmpty /> <ArticleListEmpty /> snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ArticleListEmpty />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": "Nothing found...",
      "variant": "headline",
    },
    "ref": null,
    "rendered": "Nothing found...",
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": "Nothing found...",
        "variant": "headline",
      },
      "ref": null,
      "rendered": "Nothing found...",
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
